js/form.js

```diff
@@ -0,0 +1,15 @@
+const mapFilter = document.querySelector ('.map__filters');
+const adForm = document.querySelector ('.ad-form');
+
+const disabledForm = (form) => {
```

Comment:
глагол в названии функции
```suggestion
const disableForm = (form) => {
```